boxEngine.py

```diff
@@ -0,0 +1,182 @@
+import os
+import random
+import time
+import platform
+import sys
+from configparser import ConfigParser
+from rich.console import Console
+
+console = Console()
+config = ConfigParser(comment_prefixes="#", delimiters="=")
+config.read("data/config/main.ini")
+
+# Color codes:
+def warn(stri):
+    console.print(f"[bold][bright_yellow]█ Warning! -> {stri}[/]")
+def error(stri):
+    console.print(f"[bold][bright_red]█ Error! -> {stri}[/]")
+def success(stri):
+    console.print(f"[bold][bright_green]█ Success! -> {stri}[/]")
+
+try:
+    configCheck1 = str(config["DEBUG"]["disableRem"])
+    if configCheck1 == "false":
+        print("# Reminder!! Every command is CaSe SeNsItIvE")
+    else: ...
+
+except:
+    exit("Error! Config file or an element is missing!")
+
+# cmd items not worth putting there
+cmdList = ["shellBasic","source","pyclockInit","passgenInit","help","quit","exit","vars","cvar","box"]
+ver = str(config["MAIN"]["version"])
+console.print(f"[ [bold][bright_green]OK[/] ][italic] boxEngine is loaded![/]")
+
+class boxEngine:
+
+    def box(string):
+        from git import Repo
+        repo = Repo()     
+
+    def rok():
+        return "Exit status:", 0
+    
+    def rno():
+        return "Something wrong happened, exit status:",0
+
+    def rmdir(path: str):
+        os.rmdir(path)
+
+    def mkdir(name: str):
+        os.mkdir(name)
+
+    def spamClear():
```

Comment:
Maybe not the most clean variant. It doesn't really clear the console. A cleaner approach for this variant would be:

print("\n"*100)

That way you don't need the for loop.

Reply:
> Maybe not the most clean variant. It doesn't really clear the console. A cleaner approach for this variant would be:
> 
> print("\n"*100)
> 
> That way you don't need the for loop.

Agreed, but i remade this clean function.